libs/domain/search/sort/src/sort.component.ts

```diff
@@ -29,6 +30,8 @@ export class SortComponent extends LitElement {
   protected renderSorting(
     sort: ProductListSort | undefined | null
```

Comment:
My suggestion to make check before invoke `renderSorting` method. To avoid `undefined | null`

Reply:
I keep it even more simple by using the sortingOptions directly in the method